packages/at_persistence_secondary_server/lib/src/keystore/hive_base.dart

```diff
@@ -35,7 +35,9 @@ mixin HiveBase<E> {
       }
     }
     if (getBox().isOpen) {
-      _logger.info('$boxName initialized successfully');
+      _logger.finer('$boxName initialized successfully');
```

Comment:
useless INFO message, it always opens successfully unless something is hideously wrong